devel/python/python/ert/enkf/local_obsdata.py

```diff
@@ -52,6 +52,12 @@ def addNode(self, node):
         already_exists_node_for_key = LocalObsdata.cNamespace().add_node(self, node)
         return already_exists_node_for_key
 
+    def delNode(self, key):
```

Comment:
Suggestion:
Python has an operator **__delitem__(self, key)** similiar to **__getitem__**. 

Then you can delete items like this:
**del local_obsdata[key]**  

https://docs.python.org/2/reference/datamodel.html#emulating-container-types